src/components/views/right_panel/UserInfo.tsx

```diff
@@ -1533,7 +1547,7 @@ const UserInfo: React.FC<IProps> = ({ user, room, onClose, phase = RightPanelPha
 
     const classes = ["mx_UserInfo"];
 
-    let cardState: IRightPanelCardState;
+    let cardState = undefined as unknown as IRightPanelCardState;
```

Comment:
looks like it's possible for this to end up as `undefined`, so shouldn't it have a type that reflects that?

Reply:
Tidied up by initialising to an empty object (a valid `IRightPanelCardState`) instead of using coercion here